package.json

```
@@ -11,7 +11,8 @@
     "frontend": "react-scripts start",
     "build": "react-scripts build",
     "deploy": "git pull && npm install && pm2 stop database api app && pm2 delete database api app && pm2 start mongod --name \"database\" && npm run build && pm2 start npm --name \"app\" -- start && pm2 start npm \"app\" -- run dev",
-    "lint": "./node_modules/.bin/eslint src/ api/models/ api/mailer/ api/routes/ api/util util/ test/"
+    "lint": "./node_modules/.bin/eslint src/ api/models/ api/mailer/ api/routes/ api/util util/ test/",
+    "create-user": "node ./api/CreateUserScript.js"
```

consider adding a `create-user` script in `api/package.json`, and then in this file you can do:
```suggestion
    "create-user": "npm run create-user --prefix api"
```